views/partial/footer.ejs

```diff
@@ -1,2 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>PDA MIT</title>
+    <!-- Pre-build CSS -->
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+	<link href="http://fonts.googleapis.com/css?family=Cookie" rel="stylesheet" type="text/css">
+</head>
+<body>
+
+    <!--The text and break tags below are placed just for testing purpose. They can be removed later-->
+    <h1>Scroll down to view the footer</h1><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
+
+    <footer class="footer-distributed">
+
+        <div class="footer-left">
+    
+            <img src="public\img\pda-logo.png" alt="logo" class="logo">
+            
+            <h3>PDA</h3>
+    
+            <p class="footer-company-about">                
+                The Personality Development Association of MIT is an interdisciplinary
+    environment that allows similar yet different students to come together
+    and help each other develop soft skills and industrial maneuvers. Relevantly,
+    our motto is “Discover Yourself”.
+            </p>
+                        
+        </div>
+
+        <div class="footer-center-1">
+            <h3>Quick links</h3>
+            <ul class="footer-links">
+                <li>
+                    <a href="#">Home</a>
+                </li>
+
+                <li>
+                    <a href="#">Events</a>
+                </li>
+
+                <li>
+                    <a href="#">About</a>
+                </li>
+
+                <li>
+                    <a href="#">Login</a>
+                </li>
+
+                <li>
+                    <a href="#">Contact</a>
+                </li>
+            </ul>
+            
+        </div>
+    
+        <div class="footer-center-2">
+            
+            <h3>Get in Touch</h3>
```

Comment:
Place some gap below the title **Get in Touch**

![image](https://user-images.githubusercontent.com/53329034/147415594-37e6d02b-2807-437c-ab89-ad5287116255.png)